pages/linux/dmidecode.md

```diff
@@ -15,10 +15,14 @@
 
 `sudo dmidecode -s system-serial-number`
 
-- Show all the BIOS information:
+- Show BIOS information:
```

Comment:
Perhaps we could be a bit more description as to the kind of things that the BIOS information might contain? I'm not sure.

Reply:
Things like which BIOS features are available - PCI, PNP, vendor name, vendor version. Do you want to add these ?

Reply:
Perhaps `(includes BIOS features and system information)` or something?

Reply:
Hmm .. actually system information has another tag `system`. :smile:  Do you want me to include both tags in the example ?

Reply:
Ah, let's leave it then :P